frontend/src/toaster.tsx

```diff
@@ -38,7 +38,7 @@ class Toaster extends Logger {
       await sleep(2000);
     }
 
-    this.node = instance.return.return;
+    this.node = instance.sibling.child;
```

Comment:
incorrect merge, should be
```suggestion
    this.node = instance.return.return;
```